storage/Devices/BcacheImpl.cc

```diff
@@ -615,29 +615,19 @@ namespace storage
 
 	if(get_type() == BcacheType::BACKED)
 	{
-	    const BlkDevice* blk_device = get_backing_device();
+	    const BlkDevice* backing_device = get_backing_device();
```

Comment:
Please add a null check here to avoid a crash if for some reason there is no backing device.

Reply:
There is already a check for that, `get_backing_device` method throws an `DeviceNotFound` exception when there is no backing device.

Reply:
OK.